AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryErrorTrackerProxyImpl.kt

```diff
@@ -0,0 +1,41 @@
+package com.automattic.android.tracks.crashlogging.internal
+
+import android.content.Context
+import io.sentry.Sentry
+import io.sentry.SentryEvent
+import io.sentry.SentryOptions
+import io.sentry.android.core.SentryAndroid
+import io.sentry.protocol.User
+
+class SentryErrorTrackerProxyImpl : SentryErrorTrackerProxy {
```

Comment:
Nitpick 🔍 : I think we usually call classes which just delegate calls to the original object "wrappers" instead of "proxies". Having said that, I agree it's a proxy so I'm fine keeping the name.

Reply:
If it can help anyone later understand this class, I'm cool with changing it to widely used `Wrapper` naming. Updated in [`9f6f595` (#51)](https://github.com/Automattic/Automattic-Tracks-Android/pull/51/commits/9f6f595f619410a9509a20ef17f3fe70a46989aa)

Reply:
Thanks!